geopyspark/geotrellis/geotiff_rdd.py

```diff
@@ -72,13 +73,24 @@ def get(geopysc,
         options = kwargs
 
     if options:
-        srdd = geotiff_rdd.get(geopysc.sc,
-                               key,
-                               uri,
-                               options)
+        if type(uri) == list:
```

Comment:
It might be better to use `isinstance(uri, list)` since that would return `True` for `list`s and any of its subclasses.